src/app/features/StepDefinitionEditor/Models/MockDataInstanceModel.js

```diff
@@ -46,6 +46,11 @@ var createMockDataInstanceModelConstructor = function (
                 get: function () {
                     return toAST.call(this);
                 }
+            },
+            link: {
```

Comment:
Shouldn't need to use this here. You should be able to directly create this link from the template by using `ui-sref="tractor.mockData({ file: mockData })"`